index.html

```diff
@@ -43,23 +43,20 @@
         <dl class="info-headings">
           <div>
             <dt>Data Gathered Since</dt>
-              <!-- @todo get date-start from store -->
-            <dd id="date-start">
-              <time datetime="2017-06-13T11:00">June 13, 2017</time>
+            <dd>
+              <time id="data-gathered-since" datetime=""></time>
             </dd>
           </div>
           <div>
             <dt>You Have Visited</dt>
-            <!-- @todo get num-sites-visited from store -->
             <dd>
-              <var id="num-sites-visited">6</var> Sites
+              <var id="num-first-parties"></var>
             </dd>

```

Comment:
```
<var><span id="num-first-parties"></span> sites</var>
```

Reply:
Just that my understanding of the semantics are that var tags should just have data rather than sentences etc.

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/var << represents a variable in the mathematical sense. It's either wrong to use at all or better being how we have it.